scripts/pipelines/gcloud/templates/package/package-pipeline.cfg

```
@@ -1,8 +1,10 @@
 #!/bin/bash
 # Mandatory flags
 mandatoryFlags="${pipelineName},$imageName,$buildPipelineName,${language}${dockerFile},$qualityPipelineName,"
-[ $language == "python" ] && mandatoryFlags=$mandatoryFlags"${languageVersion},"
-[ $language == "flutter" ] && mandatoryFlags=$mandatoryFlags"${registryLocation},${languageVersion},${flutterPlatform},"
+[ "$language" == "python" ] && mandatoryFlags=$mandatoryFlags"${languageVersion},"
+# Check if any flutter platform is defined
+[[ -n "$androidPlatform" || -n "$webPlatform" ]] && flutterPlatformDefined=true
+[ "$language" == "flutter" ] && mandatoryFlags=$mandatoryFlags"${registryLocation},${languageVersion},${flutterPlatformDefined},"
```

```suggestion
[ "$language" == "flutter" ] && mandatoryFlags=$mandatoryFlags"${registryLocation},${languageVersion},${webPlatform}${androidPlatform},"
```

Shouldm't this be like this? Ask @ultymatom in case of doubt.

---

yes that would work too
that would mean, if webPlatform AND androidP?latform are empty it will fail
but if one (or two) of them is not empty it will pass